app/src/main/java/fr/neamar/kiss/result/Result.java

```diff
@@ -168,6 +165,11 @@ ListPopup inflatePopupMenu(ArrayAdapter<ListPopup.Item> adapter, Context context
             }
         }
 
+        if (BuildConfig.DEBUG) {
+            int relevance = pojo.relevance - 1;
```

Comment:
Any reason for the `- 1` ?
It was there before, granted, but I don't see why we need it

Reply:
I don't know, @TBog maybe?